unlock-app/package.json

```diff
@@ -42,7 +42,7 @@
     "react-dom": "^16.6.3",
     "react-jazzicon": "^0.1.2",
     "react-no-ssr": "^1.1.0",
-    "react-redux": "^6.0.0",
+    "react-redux": "^5.0.7",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.9",
```

Comment:
```suggestion
    "react-router-redux": "^4.0.8",
```

Reply:
also, and more importantly

connect-react-router must be at version 5.0.1 not 6.0 beta. 6.0 requires react-redux 6

Reply:
github won't let me suggest that change